spacy/language.py

```diff
@@ -1727,12 +1729,6 @@ def from_config(
 
         DOCS: https://spacy.io/api/language#from_config
         """
-        if isinstance(disable, str):
-            disable = [disable]
-        if isinstance(enable, str):
-            enable = [enable]
-        if isinstance(exclude, str):
-            exclude = [exclude]
```

Comment:
The checks need to be modified in order to be able to know whether an argument was passed to `spacy.load()`. This and the new approach to handling conflicting values from the config and passed arguments are implemented in `fetch_pipes_status()`.

Reply:
I don't understand how modifying this here makes a difference in the later checks?

Reply:
`fetch_pipes_status()` changes the behavior w.r.t. `disable`/`enable`/`disabled`/`enabled` values. It only uses `disabled`/`enable` from the config if `disable`/`enable` are not set at all (hence the [check for an empty `SimpleFrozenList`](https://github.com/explosion/spaCy/pull/11459#discussion_r965884449)). Otherwise only `disable`/`enable` are used. Before this PR we merged `disable` with `disabled, `enable` with `enabled` and passed all values to `_resolve_component_status()`, which raised the issue that motivated this change.

I integrated the list conversion you commented on into `fetch_pipes_status()`. I could also pull it out from there and reinstate
```python
        if isinstance(disable, str):
            disable = [disable]
        if isinstance(enable, str):
            enable = [enable]
        if isinstance(exclude, str):
            exclude = [exclude]
```
\- it shouldn't make a difference though, as far as I can tell. 

Does this answer your question?

Reply:
I think it would be easier to follow what's going on with the types if you handle the `str` type initially as it was before and then simplify the types for `fetch_pipes_status`. To me this change makes it seem like you need to know whether it was originally `str` later when you don't.

Reply:
Restored the original type checks/list conversions and simplified types for `fetch_pipes_status()`. Feel free to reopen if you think something's still amiss.